python/cuml/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -698,7 +698,7 @@ class RandomForestRegressor(BaseRandomForestModel,
 
         # shortcut for default accuracy metric of r^2
         if self.accuracy_metric == "r2":
-            stats = r2_score(y_m, preds, handle=self.handle)
+            stats = r2_score(y_m, preds)
```

Comment:
I _think_ this change is ok (and we might not even need the `handle.sync` below?). Flagging for someone else to double-check here.

Reply:
If we are no longer calling into RAFT, then we shouldn't need the handle here.